Automattic/woocommerce-payments
# Update Dev/Sandbox Mode Choice Design

Fixes #7731

#### Changes proposed in this Pull Request






* Define new component, OnboardingCard to represent a card containing content on the onboarding flow.
* Remove RadioCard component which is no longer used.
* Update to match latest designs.

#### Screenshots

<img width="1188" alt="Screenshot 2024-01-10 at 11 53 19" src="https://github.com/Automattic/woocommerce-payments/assets/11770181/7baf2582-35d3-49d3-8cd0-6f237aeea42a">

Should match the design as laid out here: paJDYF-bGl-p2
 
Note: I haven't removed the subheading on the design yet as I wanted to double check this first.





#### Testing instructions





* Disconnect or delete your previous account in the DB so WooPayments thinks there is no account connected.
* Initiate a new onboarding flow.
* Verify that the new design for the first page of the flow looks as expected.
* Start the flow using the "live" option (if you have sandbox mode enabled it will still create a test account) and go through the flow and verify that you can create an account.
* Start the flow using the sandbox option and verify you get redirected to the Stripe test KYC and you can create an account.

-------------------


**Post merge**
